### DISCUSSION TITLE
Declaring an unmanaged delegate* type doesn't seem to be possible

### DISCUSSION THREAD

**Discussion by Rosentti** [dotnet/csharplang#7540] [Q&A] [answered]
With managed delegates you can do:
`public delegate byte LessFunc_t(K* first, K* second);`
However this isn't good enough since you can't define pointers to it. 
This declares a field:
`delegate* unmanaged[Cdecl]<K*, K*, byte> LessFunc_t;`
So how to do a Type? Is it possible? I can't wrap my head around the syntax..

**Comment by 333fred** [comment, marked as answer]
C# 12 will allow you to make a global using alias for pointer types, like `global unsafe using MyFunc = delegate*<void>;`. Outside of that, there is no way to declare a specific name for a function pointer definko, just like there is no way to declare a specific name for any other pointer type.

**Reply by Rosentti** [reply]
:(
At least it's coming though.
